modules/json/filterx-cache-json-file.c

```diff
@@ -124,7 +124,8 @@ _load_json_file(const gchar *filepath, GError **error)
   if (object)
     {
       result = filterx_json_new_from_object(object);
-      filterx_object_make_readonly(result);
+      if (result)
+        filterx_object_make_readonly(result);
```

Comment:
If we don't get a `FilterXObject` here, we should free `object`, or it will be leaked.